actors/builtin/miner/miner_actor.go

```diff
@@ -498,25 +497,31 @@ func (a Actor) PreCommitSector(rt Runtime, params *SectorPreCommitInfo) *adt.Emp
 
 	notifyPledgeChanged(rt, newlyVested.Neg())
 
-	bf := abi.NewBitField()
-	bf.Set(uint64(params.SectorNumber))
-
-	// Request deferred Cron check for PreCommit expiry check.
-	cronPayload := CronEventPayload{
-		EventType: CronEventPreCommitExpiry,
-		Sectors:   bf,
-	}
-
+	// add precommit expiry to the queue
 	msd, ok := MaxSealDuration[params.SealProof]
 	if !ok {
 		rt.Abortf(exitcode.ErrIllegalArgument, "no max seal duration set for proof type: %d", params.SealProof)
 	}
-
 	// The +1 here is critical for the batch verification of proofs. Without it, if a proof arrived exactly on the
 	// due epoch, ProveCommitSector would accept it, then the expiry event would remove it, and then
 	// ConfirmSectorProofsValid would fail to find it.
 	expiryBound := rt.CurrEpoch() + msd + 1
-	enrollCronEvent(rt, expiryBound, &cronPayload)
+
+	// create BitField for this sector
+	bf := abi.NewBitField()
+	bf.Set(uint64(params.SectorNumber))
+
+	// Load BitField Queue for sector expiry
+	quant := st.QuantEndOfDeadline()
```

Comment:
This has gone away. We'll need to quantize for every deadline, at the end of every deadline.

Reply:
`NewQuantSpec(WPoStChallengeWindow, st.ProvingPeriodStart)`